packages/demo/useCryptKeeper.ts

```diff
@@ -216,23 +227,24 @@ export const useCryptKeeper = (): IUseCryptKeeperData => {
   }, [client]);
 
   const getConnectedIdentity = useCallback(async () => {
-    const payload = await client?.getConnectedIdentity();
-
-    if (!payload) {
-      return;
+    try {
+      const fetchedConnectedIdentityMetadata = await client?.getConnectedIdentity();
+      setConnectedIdentityMetadata(fetchedConnectedIdentityMetadata);
+      toast(
+        `Getting Identity Metadata Successfully! ${JSON.stringify(Object.entries(fetchedConnectedIdentityMetadata!))}`,
+        {
+          type: "success",
+        },
+      );
+    } catch (error) {
+      toast(`${error as string}`, {
+        type: "error",
+      });
     }
-
-    setConnectedIdentityMetadata(payload as unknown as ConnectedIdentityMetadata);
-
-    toast(`Getting Identity data successfully!`, { type: "success" });
   }, [client, setConnectedIdentityMetadata]);
 
-  const createIdentity = useCallback(() => {
-    client?.createIdentity({ host: window.location.origin });
-  }, [client]);
-
-  const connectIdentity = useCallback(async () => {
-    await client?.connectIdentity({ host: window.location.origin });
+  const createIdentity = useCallback(async () => {
+    // await client?.createIdentity();
```

Comment:
Why is it commented?

Reply:
For the `createIdentity()` feature, it would be removed and it would be just `connectIdentity()` and based on this issue feature (https://github.com/CryptKeeperZK/crypt-keeper-extension/issues/898) the user would be able to change the current connected identity directly from the extension which will emit an event for that. Or create a new identity directly from the extension and then change the connected identity also directly from the extension.

Reply:
Ok, let's remove the commented code.